### ISSUE TITLE
WINETRICKS environment variable is not available. Searching from $PATH.

### ISSUE THREAD

**Issue by Hyphaed** [Matoking/protontricks#327]
If I write down winetricks on terminal it opens winetrciks, yet I get this error

Please include this entire error message when making a bug report.
Log messages:

Running inside Flatpak sandbox, version 1.14.6.
Found Steam directory at /home/ferran/.local/share/Steam
Using default Steam Runtime at /home/ferran/.local/share/Steam/ubuntu12_32/steam-runtime
WINETRICKS environment variable is not available. Searching from $PATH.
Found 1 Steam library folders
Currently logged-in Steam user: ferran_bcn
Found 0 Steam shortcuts running using Steam compatibility tools
Using 'yad' as GUI provider

**Comment by Matoking** [comment]
That doesn't seem to be the entire error message? The error dialog starts with the error that caused Protontricks to crash, followed by the log messages leading up to the crash. This bug report only contains the latter part.

**Comment by tamodolo** [comment]
This is what I get after selecting a game using yad

```
Protontricks was closed due to the following error:

Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/protontricks/cli/util.py", line 159, in wrapper
    return cli_func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/protontricks/cli/main.py", line 277, in main
    proton_app = find_proton_app(
                 ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/protontricks/steam.py", line 848, in find_proton_app
    tool_app = find_steam_compat_tool_app(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/protontricks/steam.py", line 609, in find_steam_compat_tool_app
    appinfo_sections = [
                       ^
  File "/usr/lib/python3.12/site-packages/protontricks/steam.py", line 531, in iter_appinfo_sections
    raise SyntaxError(
SyntaxError: Invalid file magic number. The appinfo.vdf version might not be supported by the current version of Protontricks - please check for updates.


=============

Please include this entire error message when making a bug report.
Log messages:

Found Steam directory at /home/tamodolo/.local/share/Steam
Using default Steam Runtime at /home/tamodolo/.local/share/Steam/ubuntu12_32/steam-runtime
WINETRICKS environment variable is not available. Searching from $PATH.
Found 1 Steam library folders
Currently logged-in Steam user: tamodolo
Couldn't find custom shortcuts. Maybe none have been created yet?
Using 'yad' as GUI provider
```

**Comment by Matoking** [comment]
> This is what I get after selecting a game using yad
> 
> ```
> Protontricks was closed due to the following error:
> 
> Traceback (most recent call last):
>   File "/usr/lib/python3.12/site-packages/protontricks/cli/util.py", line 159, in wrapper
>     return cli_func(self, *args, **kwargs)
>            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/usr/lib/python3.12/site-packages/protontricks/cli/main.py", line 277, in main
>     proton_app = find_proton_app(
>                  ^^^^^^^^^^^^^^^^
>   File "/usr/lib/python3.12/site-packages/protontricks/steam.py", line 848, in find_proton_app
>     tool_app = find_steam_compat_tool_app(
>                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/usr/lib/python3.12/site-packages/protontricks/steam.py", line 609, in find_steam_compat_tool_app
>     appinfo_sections = [
>                        ^
>   File "/usr/lib/python3.12/site-packages/protontricks/steam.py", line 531, in iter_appinfo_sections
>     raise SyntaxError(
> SyntaxError: Invalid file magic number. The appinfo.vdf version might not be supported by the current version of Protontricks - please check for updates.
> 
> 
> =============
> 
> Please include this entire error message when making a bug report.
> Log messages:
> 
> Found Steam directory at /home/tamodolo/.local/share/Steam
> Using default Steam Runtime at /home/tamodolo/.local/share/Steam/ubuntu12_32/steam-runtime
> WINETRICKS environment variable is not available. Searching from $PATH.
> Found 1 Steam library folders
> Currently logged-in Steam user: tamodolo
> Couldn't find custom shortcuts. Maybe none have been created yet?
> Using 'yad' as GUI provider
> ```
> 
> I'm using Arch package.

That is issue #304. There's a beta version that has the missing support, see [here](https://github.com/Matoking/protontricks/issues/304#issuecomment-2220599470) for instructions.

As for why there isn't a proper new version with this fix: one of the software libraries that Protontricks relies on needs to be fixed, but the current maintainer is inactive and there hasn't been a new release.

**Comment by tyrsalt** [comment]
I get a similar error message when i try to open a game through ProtonTricks.

Protontricks was closed due to the following error:

Traceback (most recent call last):
  File "/app/lib/python3.11/site-packages/protontricks/cli/util.py", line 159, in wrapper
    return cli_func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/protontricks/cli/main.py", line 277, in main
    proton_app = find_proton_app(
                 ^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/protontricks/steam.py", line 848, in find_proton_app
    tool_app = find_steam_compat_tool_app(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/protontricks/steam.py", line 609, in find_steam_compat_tool_app
    appinfo_sections = [
                       ^
  File "/app/lib/python3.11/site-packages/protontricks/steam.py", line 609, in <listcomp>
    appinfo_sections = [
                       ^
  File "/app/lib/python3.11/site-packages/protontricks/steam.py", line 531, in iter_appinfo_sections
    raise SyntaxError(
SyntaxError: Invalid file magic number. The appinfo.vdf version might not be supported by the current version of Protontricks - please check for updates.


=============

Please include this entire error message when making a bug report.
Log messages:

Running inside Flatpak sandbox, version 1.14.4.
Found Steam directory at /home/deck/.local/share/Steam
Using default Steam Runtime at /home/deck/.local/share/Steam/ubuntu12_32/steam-runtime
WINETRICKS environment variable is not available. Searching from $PATH.
Found 2 Steam library folders
Currently logged-in Steam user: tyrsalt
Shortcut Shattered - Tale of the Forgotten King (2905995727) does not have a prefix. It's either not a Proton app or it hasn't been launched yet.
Shortcut Cave Story+ (3801484219) does not have a prefix. It's either not a Proton app or it hasn't been launched yet.
Shortcut GRIME (2295145240) does not have a prefix. It's either not a Proton app or it hasn't been launched yet.
Shortcut Vampyr (2717290197) does not have a prefix. It's either not a Proton app or it hasn't been launched yet.
Shortcut Google Chrome (3410522567) does not have a prefix. It's either not a Proton app or it hasn't been launched yet.
Found 0 Steam shortcuts running using Steam compatibility tools
Using 'yad' as GUI provider

**Comment by Matoking** [comment]
Closing issue as this was fixed by 1.12.0.
